dist/tools/pyterm/pyterm

```diff
@@ -24,10 +24,20 @@ try:
 except ImportError:
     import ConfigParser as configparser
 
-
 import cmd, serial, socket, sys, threading, readline, time, logging, \
     os, argparse, re, codecs, signal
 
+try:
+    serial.Serial
+except AttributeError:
+    print("\033[1;37;41m\n")
```

Comment:
you can use a single print with a multi line string.

Reply:
Would not be better here I think.

Reply:
I also think that several `print`s are nicer looking here. @aabadie do you feel strongly about this? If not, then I'd like to go ahead with merging it as is.

Reply:
Not totally convinced by the argument. Here is what I was thinking about:
```python
print("\033[1;37;41m\n"
      "Something went wrong when loading the pyserial package.\n"
      "There is a good chance that you installed the 'serial' package instead"
      "of 'pyserial'.\nTry to run 'pip uninstall serial && pip install pyserial'"
      "\033[0m")
```

Reply:
@aabadie is your change rather cosmetic, i.e., your personal preferred coding style, **or** does it have any other advantages like PEP something conformity? 

If the latter does not apply, please note the discussion on the mailing list regarding blocking PRs or requesting changes that do not improve the PR quality considerably. Btw. this only a minor but still helpful enhancement to a tool used in RIOT, it does not in anyway impact performance.

Reply:
and if it's the former ? we don't care I guess...
the former is common usage of strings in Python print function. Why using multiple calls to the same functions when this can be done in one call ?

In the end I have no strong opinion on that, just wanted to show alternatives. Since 2 other maintainers approve, I'll dismiss my review.